### ISSUE TITLE
Outdated Axios Peer-Dependency

### ISSUE THREAD

**Issue by reformat0r** [dmtrbrl/vue-trend-chart#32]
Hey!
I noticed that the package lists an outdated version of `axios` as a peer-dependency.
Since some time now npm will install (and thus require) those by default - meaning that having `vue-trend-chart` anywhere in the dependency-tree prevents users from updating `axios` altogether.

```
npm ERR! Found: axios@0.21.1
npm ERR! node_modules/axios
npm ERR!   dev axios@"^0.21.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer axios@"^0.18.0" from vue-trend-chart@0.15.3
npm ERR! node_modules/vue-trend-chart
npm ERR!   vue-trend-chart@"^0.15.3" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

Workaround:
```
npm install axios@latest --save --legacy-peer-deps
```

Since `axios` seems to only be included for the demo I'd suggest creating a separate `package.json` for that.

Cheers!

**Comment by dmtrbrl** [comment]
Fixed in v0.17.0

**Comment by malles** [comment]
Could this please be done for the Vue2 version 0.15.3 as well?
